terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -1230,7 +1230,7 @@ module "dms_nomis_ingestor" {
   dms_target_name              = "kinesis"
   short_name                   = "nomis"
   migration_type               = "full-load-and-cdc"
-  replication_instance_version = "3.4.7" # Upgrade
+  replication_instance_version = "3.5.2"
```

Comment:
I've changed this because 3.4.7 is no longer available and so breaks the terraform apply